src/main/java/duke/Geng.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import duke.commands.Command;
+import duke.tasks.TaskList;
+import duke.ui.*;
+
```

Comment:
Consider renaming the package duke to something more relevant, such as geng, to align with the application's current name. This would improve clarity and consistency with the project. Perhaps import your ui classes one by one to follow the coding standard
```suggestion
package dnar;

import dnar.commands.Command;
import dnar.tasks.TaskList;
import dnar.ui.''
import dnar.ui.''
import dnar.ui.''
```